atd-vze/src/_nav.js

```diff
@@ -48,21 +48,14 @@ export const navigation = roles => {
       icon: "fa fa-heart-o",
     },
     {
-      name: "Tools",
-      url: "#",
-      icon: "icon-wrench",
-      children: [
-        {
-          name: "Upload Non-CR3",
-          url: "/tools/upload_non_cr3",
-          icon: "icon-cloud-upload",
-        },
-        {
-          name: "Create Crash Record",
-          url: "/tools/create_crash_record",
-          icon: "icon-shield",
-        },
-      ],
```

Comment:
cool—i think it's fine that the route is changing 👍

Reply:
cool yeah i figured why not :shrug: